### PR TITLE
Fix up .bldr.toml doc

### PR DESCRIPTION
**Pull request by chefsalim**
Update the docs for .bldr.toml

Signed-off-by: Salim Alam <salam@chef.io>

tenor-117611622

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/partials/docs/_dev-pkgs-multiple-plans-builder.html.md.erb [hunk 1]
```diff
@@ -12,22 +12,25 @@ To enable this functionality, do the following:
...
 
-    The `.bldr.toml` file is in TOML format, so create a TOML table specifying the `$pkg_name` value for that plan and then add a `path` field specifying Unix-style path globs to files or directories within your project. File or directory changes made in these path locations determine which packages will be rebuilt.
+    The `.bldr.toml` file is in TOML format, so create a TOML table specifying the `$pkg_name` value for that plan and then add a `plan_path` field specifying the path to your `plan.sh` file (you do not need to include plan.sh explicitly in the path). If all the files related to the plan are under the plan path, then you are done. Otherwise, you will need an addition 'path' field specifying Unix-style path globs to files that are associated with the plan you specified in the 'plan_path'. File or directory changes made in these path locations determine which packages will be rebuilt. Basically, when a file is committed, Builder will check to see whether it falls underneath the `plan_path` hierarchy, or matches one of the globs in the `path` field if it was specified - if the answer is yes, then Builder will issue a build for that commit.
```
Comment (cnunciato) [inline www/source/partials/docs/_dev-pkgs-multiple-plans-builder.html.md.erb]: `additional path` path

**Comment by tashimi** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/317181948).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/317181948).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

File: www/source/partials/docs/_dev-pkgs-multiple-plans-builder.html.md.erb [hunk 2]
```diff
@@ -47,3 +50,9 @@ You can also specify a list of additional channels that packages should automati
 [hab-launcher]
 channels = [stable"]
```
Comment (bixu) [inline www/source/partials/docs/_dev-pkgs-multiple-plans-builder.html.md.erb]: Is this meant to be
```
channels = ["stable"]
```
?

**Pull request merged**